README.md

```diff
@@ -9,8 +9,10 @@ The editor is fully guided. Features include:
 * custom target image
 * free target zone positioning and sizing
 * custom zone labels
+* ability to show or hide zone borders
 * custom text and background colors for items
 * image items
+* items prompting for additional (numerical) input after being dropped
```

Comment:
I'm actually hoping we can remove this feature in the medium term... since I don't think it will be used much (ever?) and it adds a lot of complexity to the code. So I'd rather not mention it. In my current PR, I've actually hidden the settings for it behind a "Show advanced settings" link. What do you think though?